app/src/androidTest/java/net/skyscanner/backpack/BpkSnapshotTest.kt

```diff
@@ -18,30 +18,23 @@
 
 package net.skyscanner.backpack
 
-import android.content.Context
 import android.os.Looper
-import android.view.ContextThemeWrapper
 import android.view.View
 import androidx.test.platform.app.InstrumentationRegistry
 import com.facebook.testing.screenshot.Screenshot
 import com.facebook.testing.screenshot.ViewHelpers
 import com.facebook.testing.screenshot.internal.TestNameDetector
-import net.skyscanner.backpack.demo.R
-import net.skyscanner.backpack.util.BpkTheme
 
 open class BpkSnapshotTest {
 
   private var height = 100
   private var width = 100
-  var testContext: Context =
-    BpkTheme.wrapContextWithDefaults(
-      ContextThemeWrapper(
-        InstrumentationRegistry.getInstrumentation().targetContext,
-        R.style.AppTheme
-      )
-    )
+
+  private val variant = BpkTestVariant.current
+  var testContext = variant.newContext(InstrumentationRegistry.getInstrumentation().targetContext)
 
   protected fun setupView(view: View) {
+    view.layoutDirection = testContext.resources.configuration.layoutDirection
```

Comment:
View doesn't use the layout direction from resources by default and inherits it from its parent.